### DISCUSSION TITLE
adding pages

### DISCUSSION THREAD

**Discussion by SHADOWELITE7** [community/community#21872] [Other Feature Feedback, Questions, & Ideas]
<p>So I am doing this in Atom and GitHub desktop. I was wondering when you do the <a href="http://user.github.io" rel="nofollow noopener">user.github.io</a> link and you have another repo and want to add it to a GitHub page. Do you put the repo as a page or do you make another file in the <a href="http://user.github.io" rel="nofollow noopener">user.github.io</a> repo</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/shadowelite7">@shadowelite7</a></p>
<p>What do you mean when you say that you want to make another repository into a page? Are you trying to link to another repository or is the other repository also a Pages site that you want to combine?</p>
<p>Thanks!</p>

**Comment by SHADOWELITE7** [comment]
<p>Want to add like<br>
If you look at steam and hover on steam username it will give you a list of pages<br>
That’s what I and trying to ask</p>

**Comment by ThirdEyeRose** [comment]
<p>I’m sorry, I’m still not familiar with what you’re referring to. Could you take a screenshot of what you’re referring to and want to replicate?</p>
<p>Thanks!</p>

**Comment by SHADOWELITE7** [comment, marked as answer]
<p>found my answer during web design class, Nav Bar</p>
